server/apps/publish/formatters/newsml_1_2_formatter.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import xml.etree.ElementTree as etree
+from xml.etree.ElementTree import SubElement
+from apps.publish.formatters import Formatter
+import superdesk
+from superdesk.errors import FormatterError
+from superdesk.utc import utcnow
+
+
+class NewsML12Formatter(Formatter):
+    """
+    NewsML 1.2 Formatter
+    """
+    XML_ROOT = '<?xml version="1.0"?><!DOCTYPE NewsML SYSTEM "http://www.aap.com.au/xml-res/NewsML_1.2.dtd">'
+    USAGE_TYPE = 'AAP content is owned by or licensed to Australian Associated Press Pty Limited (AAP) and is ' \
+                 'copyright protected.  AAP content is published on an "as is" basis for personal use only and ' \
+                 'must not be copied, republished, rewritten, resold or redistributed, whether by caching, ' \
+                 'framing or similar means, without AAP\'s prior written permission.  AAP and its licensors ' \
+                 'are not liable for any loss, through negligence or otherwise, resulting from errors or ' \
+                 'omissions in or reliance on AAP content.  The globe symbol and "AAP" are registered trade marks. ' \
+                 'Further this AAP content is supplied to the direct recipient pursuant to an Information Supply ' \
+                 'Agreement with AAP (AAP Information Supply Agreement).  The direct recipient has a non-exclusive, ' \
+                 'non-transferable right to display this AAP content in accordance with and subject to the ' \
+                 'terms of the AAP Information Supply Agreement.'
+    LIMITATIONS = 'The direct recipient must comply with the limitations specified in the AAP Information ' \
+                  'Supply Agreement relating to the AAP content including, without limitation, not permitting ' \
+                  'redistribution and storage of the content (outside the terms of the Agreement) and not ' \
+                  'permitting deep hyperlinking to the content, framing of the content on a web site, ' \
+                  'posting the content to usenet newsgroups or facilitating such actions.'
```

Comment:
You mean the triple quotes? That's an option, too, indeed.
Perhaps its only downside is that it considers all indentation whitespace as a part of the string, which is probably not desirable for a human-readable text.